web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -436,9 +464,109 @@ export class BarChartComponent implements AfterViewInit {
       .extent([[70, 0], [this.width - 90, this.heightBrush]]) // Limit the brush.
       .on('brush', brushDown); // Update the focus bar chart based on the brush selection.
 
-    this.brush.append('g')
+    this.brush
+      .append('g')
       .attr('class', 'brush')
       .call(brushSelector)
       .call(brushSelector.move, [firstItemPosition, lastItemPosition]);
   }
+
+  /**
+   * This function creates a boxplot next to the focus bar chart with all selected data.
+   * It indicates maximum, minimium, median, first quantile and third quantile of the
+   * selected data durations.
+   *
+   * @param inputData: Data selected for the focus bar chart
+   */
+  private createBoxplot(inputData: Item[]): void {
+    // Remove all elements of from the previous boxplot,
+    if (!this.boxplot) { return; }
```

Comment:
```suggestion
    if (!this.boxplot) { return; }
    // Remove all elements of from the previous boxplot,
```
Move the comment closer to the matching code.